### PR TITLE
Add R random forest example

### PR DESCRIPTION
**Pull request by chrix-sigopt**
cc @iandewancker @alexandraj777

### CODE REVIEW

File: R-random-forest/random_forest_SigOpt.R [hunk 1]
```diff
@@ -0,0 +1,82 @@
+# install packages
+install.packages("devtools", repos = "http://cran.us.r-project.org")
+library(devtools)
+install_github("sigopt/SigOptR")
+library(SigOptR)
+install.packages('randomForest', repos = "http://cran.us.r-project.org")
+library(randomForest)
+
+# set SigOpt client token
+# get your client API token here : https://sigopt.com/user/profile
```
Comment (alexandraj777) [inline R-random-forest/random_forest_SigOpt.R]: I use this line in other examples:
`# Learn more about authenticating the SigOpt API: sigopt.com/docs/overview/authentication`

Comment (chrix-sigopt) [inline R-random-forest/random_forest_SigOpt.R]: Fixed!

**Comment by chrix-sigopt** [comment]
Do we need to worry about Travis in this repo?

**Comment by pfhayes** [comment]
No, sorry about that - you can ignore that

**Comment by alexandraj777** [comment]
I asked @pfhayes about this when I saw the failures, Travis isn't currently up-and-running for this repo so no need to worry about it :-)

**Comment by alexandraj777** [comment]
I was one second too slow with my comment

**Pull request merged**